sympy/stats/crv_types.py

```diff
@@ -161,6 +161,14 @@ def rv(symbol, cls, args):
 class ArcsinDistribution(SingleContinuousDistribution):
     _argnames = ('a', 'b')
 
+    @staticmethod
+    def check(a, b):
+        _value_check(a.is_real and b.is_real, "Parameters should be real.")
```

Comment:
`_value_check(fuzzy_and(a.is_real, b.is_real), msg)` or `all(_value_check(i.is_real, msg) for i in (a, b))` with diff I sent to your other PR.